modules/fundamental/src/vulnerability/service/test.rs

```diff
@@ -48,6 +48,8 @@ async fn statuses(ctx: &TrustifyContext) -> Result<(), anyhow::Error> {
 
     assert_eq!(2, vuln.advisories.len());
 
+    println!("{}", serde_json::to_string_pretty(&vuln)?);
```

Comment:
did u mean println! or maybe this could also be log::debug ?

Reply:
Replaced with debug log